vyper/venom/__init__.py

```diff
@@ -86,6 +87,10 @@ def _run_passes(fn: IRFunction, optimize: OptimizationLevel) -> None:
     AlgebraicOptimizationPass(ac, fn).run_pass()
     RemoveUnusedVariablesPass(ac, fn).run_pass()
```

Comment:
i think this extra RemoveUnusedVariablesPass is now unnecessary

Reply:
resolved: https://github.com/vyperlang/vyper/pull/4241/commits/3fd3a0aa2cf8a8c630ce981950daea390a609092